main.py

```diff
@@ -1,33 +1,48 @@
 import app
-print("*************")
-print("Welcome")
-print("*************")
 
 
+user_name = input("Plese enter user name: ")
+print("*************")
+print(f"Welcome {user_name}")
+print("*************")
 
 while True:
 
-    choice = input("1 - enter task\n2 - find task\n3 - update task\n4 - delete task\n0 - exit\n")
+    choice = input("1 - enter task\n2 - find task\n3 - update by task title\n4 - delete task by name\n0 - exit\n")
```

Comment:
Check for the right type

Reply:
"match choice" requires string, and string is received from "input".
If string does not match listed values, loop starts from the beginning.